core/src/main/scala/org/apache/spark/util/logging/FileAppender.scala

```diff
@@ -26,8 +26,12 @@ import org.apache.spark.util.{IntParam, Utils}
 /**
  * Continuously appends the data from an input stream into the given file.
  */
-private[spark] class FileAppender(inputStream: InputStream, file: File, bufferSize: Int = 8192)
-  extends Logging {
+private[spark] class FileAppender(
+  inputStream: InputStream,
+  file: File,
+  bufferSize: Int = 8192,
+  closeStreams: Boolean = false
```

Comment:
shall we name it `closeInputStream`?

Reply:
Look at this again. When will we explicitly set `closeStreams` to `false`? cc @jhu-chang